doc/sphinx/language/coq-library.rst

```diff
@@ -706,18 +706,15 @@ Accessing the Type level
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 The basic library includes the definitions of the counterparts of some data-types and logical
-quantifiers at the ``Type``: level: negation, pair, and properties
+quantifiers at the ``Type``: level: negation and properties
```

Comment:
As it is now, this sentence sounds a bit odd. Maybe it could be reformulated? And what is this repetition of a colon.